src/recipes/hot-expresso.json

```
@@ -0,0 +1,16 @@
+{
+    "name": "Hot Expresso",
+    "description": "Morning boost expresso shots",
+    "github": "https://github.com/cendekia",
```

Just your github username is fine:
```suggestion
    "github": "cendekia",
```

---

ahhh.. okay I will change it